### ISSUE TITLE
Use mmtracking for text tracking

### ISSUE THREAD

**Issue by anhquan075** [open-mmlab/mmtracking#502]
Hello MMTracking team,
I want to use some datasets in video text spots to train for the video object detection task. Is it possible to do this?

**Comment by Seerkfang** [comment]
Yes. But you may modify the size of anchors according to the traits of the text.
